### ISSUE TITLE
Issue with Model File Not Saving After Training

### ISSUE THREAD

**Issue by hellcatmon** [anhnh2002/XTTSv2-Finetuning-for-New-Languages#29]
Hi, 
I am encountering an issue where the model file is not being saved after completing the training process. This has occurred multiple times, whether I use the same dataset or different ones. Any thoughts or ideas on how to resolve this issue would be greatly appreciated.

Attached is the log file for your reference. Below is the command I have been using to execute the training:

```shell
!CUDA_VISIBLE_DEVICES=0 python train_gpt_xtts.py \
--output_path /tmp/checkpoints/ \
--metadatas /tmp/dataset-1/metadata_train.csv,/tmp/dataset-1/metadata_eval.csv,be \
--num_epochs 5 \
--batch_size 4 \
--grad_acumm 4 \
--max_text_length 500 \
--max_audio_length 5000750 \
--weight_decay 1e-2 \
--lr 5e-6 \
--save_step 50000
``` 

The last step screen

<img width="868" alt="image" src="https://github.com/user-attachments/assets/74f13149-bcf7-4579-b170-81af8151c27a">

And the log file
[trainer_0_log.txt](https://github.com/user-attachments/files/17781066/trainer_0_log.txt)

Thanks

**Comment by hellcatmon** [comment]
Please let me know if any additional data is needed.

**Comment by anhnh2002** [comment]
It has been solved at #25

**Comment by hellcatmon** [comment]
@anhnh2002 Thanks
